features/plate_destruction_verification.feature

```diff
@@ -1,33 +1,34 @@
 @sequencescape_service @user_barcode_service @audit_process @javascript @verification @timecop
+@labwhere_scan_into_destroyed_location_success
 Feature: Verify destruction of plates
 
   Background:
     Given user "john" with barcode '2470000100730' exists
-      And I have a "Plate destruction" instrument with barcode "abc123456"
-      And I have a process "Destroy plates" as part of the "Plate destruction" instrument with destroy plates verification
-      And all this is happening at "2015-05-05T13:45:00+01:00"
+    And I have a "Plate destruction" instrument with barcode "abc123456"
```

Comment:
Does asset audits only have cucumber tests?

Reply:
It has Cucumber tests and ActiveSupport::TestCase tests. I have now added tests using the latter.